diff --git a/src/Logger/Logger.php b/src/Logger/Logger.php
--- a/src/Logger/Logger.php
+++ b/src/Logger/Logger.php
@@ -5,6 +5,7 @@
 use Drupal\Core\Logger\LoggerChannelInterface;
 use Drupal\Core\Logger\RfcLogLevel;
 use Drupal\Core\Session\AccountInterface;
+use Monolog\DateTimeImmutable;
 use Monolog\Logger as BaseLogger;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
@@ -41,13 +42,14 @@
   public function addRecord(
     int $level,
     string $message,
-    array $context = []
+    array $context = [],
+    DateTimeImmutable $datetime = NULL
   ): bool {
     if (array_key_exists($level, $this->levelTranslation)) {
       $level = $this->levelTranslation[$level];
     }

-    return parent::addRecord($level, $message, $context);
+    return parent::addRecord($level, $message, $context, $datetime);
   }

   /**
